styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -416,8 +417,13 @@ public void create(Environment environment) {
 
     final Supplier<StyxConfig> styxConfig = new CachedSupplier<>(storage::config, time);
     final Debug debug = () -> styxConfig.get().debugEnabled();
+
+    Duration maxRunningStateTtl = get(config, config::getString, STYX_RUNNING_STATE_MAX_TTL_CONFIG)
```

Comment:
```suggestion
    final Duration maxRunningStateTtl = get(config, config::getString, STYX_RUNNING_STATE_MAX_TTL_CONFIG)
```
Same here, or just `var`.